app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <%= render 'shared/maintenance_window_alert' do %>
   <h1><%= t('doc_auth.headings.welcome') %></h1>
-  <p class='mt-tiny margin-bottom-0'><%= t('doc_auth.info.welcome_html') %></p>
+  <p class='mt-tiny margin-bottom-0'><%= t('doc_auth.info.welcome_html', sp_name: decorated_session.sp_name) %></p>
```

Comment:
Since `sp_name` might not always be available (e.g. local development, or after password recovery), should we keep around the old texts as a backup for those situations?

Reply:
Herm, I thought about this but was just thinking about local development...but you're right in the password recovery there wouldn't be an sp_name either.

I'll probably go with a default of "The agency" (translated) for simplicity in the rare cases where the public will encounter the page without an SP. Seem reasonable? 

@anniehirshman-gsa thoughts?

Reply:
Yeah, from my perspective, that could work 👍

Reply:
I think the current text is "The agency you are trying to access...", so if that exists with translations already, we can use it as the back-up if no SP name is enabled?

Reply:
Ok, added.